version

```diff
@@ -1,3 +1,3 @@
 Version 0.7.9
 DataVersion 0
-6.10-7.6
```

Comment:
This is already part of another PR.

Reply:
you are right. it's here by mistake. good catch